Docker.DotNet/Microsoft.Net.Http.Client/UnixDomainSocketEndPoint.cs

```diff
@@ -0,0 +1,110 @@
+#if NETSTANDARD1_6
```

Comment:
There is a license file in the same folder. Is that the right one to use?





On Thu, Aug 18, 2016 at 7:00 PM -0700, "John Starks" <notifications@github.com<mailto:notifications@github.com>> wrote:


In Docker.DotNet/Microsoft.Net.Http.Client/UnixDomainSocketEndPoint.cs<https://github.com/Microsoft/Docker.DotNet/pull/113#discussion_r75418438>:

> @@ -0,0 +1,110 @@
> +#if NETSTANDARD1_6


OK, but now the copyright statement says read the LICENSE file, which is our license file, not the .NET one.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/Docker.DotNet/pull/113/files/e51426c37f04447d116d7e1d1d128f592df573b7#r75418438>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKE-d4VkaBU2dnlqr_CgrDLOYeVxr8VLks5qhQ4dgaJpZM4Jn8ka>.